run-ci.sh

```diff
@@ -0,0 +1,20 @@
+FIRST_COMMIT_HASH=$( git log master..$( git branch --show-current ) --pretty=oneline | tail -1 | awk '{ print $1 }' )
+LAST_COMMIT_HASH=$( git rev-parse HEAD )
+ARTICLES=$( git diff --diff-filter=d --name-only ${FIRST_COMMIT_HASH}^ ${LAST_COMMIT_HASH} "wiki/**/*.md" "news/*.md" )
+
+echo -e "\n--- Binary file size check ---\n"
```

Comment:
nit: this, being the very first announcement, produces an unnecessary empty line at the beginning

Reply:
we now have a funny problem of `printf "--- Binary file size check ---\n\n"` causing a syntax error. adding the beginning newline fixes it